File: tikito-backend/src/main/java/org/tikito/dto/export/ImportExportSettings.java
package org.tikito.dto.export;

import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class ImportExportSettings {
    private @NotNull Boolean accounts;
    private @NotNull Boolean moneyTransactions;
    private @NotNull Boolean moneyTransactionGroups;
    private @NotNull Boolean securityTransactions;
    private @NotNull Boolean loans;
}

File: tikito-ui/src/app/dto/import-export-settings.ts
export class ImportExportSettings {
  accounts: boolean = true;
  moneyTransactions: boolean = false;
  moneyTransactionGroups: boolean = false;
  securityTransactions: boolean = false;
  loans: boolean = false;

  constructor(accounts: boolean,
              moneyTransactions: boolean,
              moneyTransactionGroups: boolean,
              securityTransactions: boolean,
              loans: boolean) {
    this.accounts = accounts;
    this.moneyTransactions = moneyTransactions;
    this.moneyTransactionGroups = moneyTransactionGroups;
    this.securityTransactions = securityTransactions;
    this.loans = loans;
  }
}